src/containers/ProjectContainer/TeststepContainer/index.jsx

```diff
@@ -98,13 +98,25 @@ const TeststepContainer = (props) => {
     const handleTeststepFormSubmit = (e) => {
         // submit the form
         e.preventDefault();
-        if(cat==='add') {
+        if (cat === 'add') {
             dispatch(addTeststepsRequest(teststepFormData))
         } else {
             dispatch(editTeststepsRequest(teststepFormData))
         }
     }
 
+    const setInitialTestdata = () => {
+        setTestdataFormData(p => ({
+            ...p,
```

Comment:
please check if you can destructure and use variables coming out of INITIAL_TESTDATA_FORM_DATA

Reply:
Done